gradle.properties

```diff
@@ -1,5 +1,5 @@
 #Sat, 15 Aug 2015 15:15:18 +0000
-version=1.16.2-SNAPSHOT
+version=1.17.0-SNAPSHOT
```

Comment:
@jfeltesse-mdsol bumping the version as there's a schema change. Now's a good time to get docker working, since the schema can now support all functions in zipkin